scripts/compare-branches.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ "$#" == "0" ]; then
+	echo "usage: $0 <base> <branch> [git args...]"
+    echo ""
+    echo "print commits on <base> since merge-base with branch that are not cherrypicked to <branch>"
+    echo "extra args are passed though to git log e.g. --author=jsmith or -n 100"
+	exit 1
+fi
+
+base="$1"
+shift
+branch="$1"
+shift
+
+from="$(git merge-base $base $branch)"
```

Comment:
I think you've got the quoting backwards here. The outer quotes are implied in variable assignment, but `$base` and `$branch` should be quoted.